subsys/bluetooth/shell/bt.c

```diff
@@ -76,6 +76,10 @@ static struct bt_conn_auth_info_cb auth_info_cb;
  */
 #define HCI_CMD_MAX_PARAM 65
 
+#define DEFAULT_SCAN_TIMEOUT_SEC 10
+
+static struct k_work_delayable active_scan_timeout_work;
```

Comment:
The struct should be moved to lines 223 in the `#if defined(CONFIG_BT_OBSERVER)` part. 

The reason is that if someone build the shell with `CONFIG_BT_OBSERVER=n` then this would give a warning as being unused

Reply:
Ok.